bom/pom.xml

```diff
@@ -46,9 +46,9 @@
     <spring.version>4.3.2.RELEASE</spring.version>
     <spring-boot.version>1.4.0.RELEASE</spring-boot.version>
     <truth.version>0.28</truth.version>
-    <vertx.version>3.3.3</vertx.version>
-    <vertx-proton.version>3.4.0.Beta1</vertx-proton.version>
-    <vertx-mqtt.version>3.4.0.Beta1</vertx-mqtt.version>
+    <vertx.version>3.4.0</vertx.version>
+    <vertx-proton.version>3.4.0</vertx-proton.version>
+    <vertx-mqtt.version>3.4.0</vertx-mqtt.version>
```

Comment:
We could also revert to 3.4.0.Beta1 (for which we have an approved CQ) for the time being ...